setup-container.sh

```diff
@@ -2,9 +2,6 @@
 
 DOCKER_SONIC_MGMT="docker-sonic-mgmt"
 DOCKER_REGISTRY="sonicdev-microsoft.azurecr.io:443/"
```

Comment:
Sorry for late comment.
Please do not hard-code URL, check `ansible/vars/docker_registry.yml`